src/gluonnlp/vocab/__init__.py

```diff
@@ -24,5 +24,6 @@
 from .subwords import *
 from .vocab import *
 from .elmo import *
+from .bert import *
 
-__all__ = vocab.__all__ + subwords.__all__ + elmo.__all__
+__all__ = vocab.__all__ + subwords.__all__ + elmo.__all__ + bert.__all__
```

Comment:
Please also update `docs/api/modules/vocab.rst`

Reply:
We also need to update the return type of bert.get_model to return BERTVocab 
https://github.com/dmlc/gluon-nlp/blob/master/src/gluonnlp/model/bert.py#L550